benchmarks/sparse/spmv.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import sys
+import torch
+from .utils import gen_sparse_gcs, gen_sparse_coo, Event
+
+def test_sparse_gcs(m, nnz, test_count):
+    start_timer = Event(enable_timing=True)
+    stop_timer = Event(enable_timing=True)
+
+    gcs = gen_sparse_gcs((m, m), nnz)
+    vector = torch.randn(m, dtype=torch.double)
```

Comment:
Notice that gen_sparse_gcs and gen_sparse_coo generate different data and although the sparsity of these are the same (nnz value) then the sparsity pattern will not be the same (the distribution of non-zero elements is computed differently) and the benchmark results may be biased. To get comparable benchmark results, always use the same tensor data for both gcs and coo benchmarks.

For instance, use the following approach:
```
def gen_sparse_coo_and_gcs(...):
    dense = np.random.randn(total_values)
    fills = random.sample(list(range(total_values)), total_values-nnz)

    for f in fills:
        dense[f] = fill_value
    dense = dense.reshape(shape)

    return dense.to_sparse(), dense.to_sparse_gcs()
```